src/apis/fetchCategories.ts

```diff
@@ -0,0 +1,43 @@
+import { Category } from "../types";
+
+export const fetchCategories = async (): Promise<Category[]> => {
+    return await fetch("http://localhost:3001/categories")
+        .then((response) => response.json())
+};
+
+export const fetchDeleteCategory = async (id: string) => {
+    await fetch(`http://localhost:3001/categories/${id}`, {
+        method: "DELETE",
+    }).catch((error) => {
```

Comment:
same comment catch, dont catch here because this means fails silent  " fails successfully :D"